lab_5_scrapper/scrapper.py

```diff
@@ -176,6 +374,26 @@ def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> No
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        self.article.title = str(article_soup.find(itemprop="headline").string)
+
+        date = article_soup.find(itemprop="datePublished").attrs['datetime']
+        if isinstance(date, str):
+            self.article.date = self.unify_date_format(date)
+
+        author = article_soup.find_all('strong')[-1].string
```

Comment:
Here I can't do it another way, because the author is treated like a basic text block (alongside the main text of the article). The only distinction is the <strong> tag, and it's always the last <strong> on the page